RespondingToIssuesAndPullRequests.md

```diff
@@ -45,7 +45,7 @@ Each tool has it’s own label, e.g. _fastlane_, `fastlane_core` and _gym_.
     - Repeat the last step for each of the related PRs that the user submitted.
 - After checking out a user’s code, you should always make sure that the tests are still working.
   - Run `bundle install` to make sure all dependencies are installed
-  - Use `bundle exec fastlane test` from the root directory to run all validation steps (tests, rubocop, etc)
```

Comment:
I think most users type `fastlane` (not initial capital).
And, `rubocop` is correct.
https://github.com/rubocop-hq/rubocop